lib/execution/transaction.js

```diff
@@ -78,7 +94,13 @@ class Transaction extends EventEmitter {
   }
 
   begin(conn) {
-    return this.query(conn, 'BEGIN;');
+    const setIsolationLevel = this.isolationLevel
```

Comment:
Might be  a good idea to rename this variable so that it doesn't have same name as the class method.